.github/workflows/coverage.yaml

```diff
@@ -37,20 +48,20 @@ jobs:
           path: lcov_unit.info
 
   rust-smoke-coverage:
-    if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'CICD:run-coverage')
-    timeout-minutes: 120
+    if: contains(github.event.pull_request.labels.*.name, 'CICD:run-coverage') || ${{ github.event_name }} == 'schedule'
+    timeout-minutes: 240
     runs-on: high-perf-docker
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
       - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
       - run: rustup component add llvm-tools-preview
-      - uses: taiki-e/install-action@6f1ebcd9e21315fc37d7f7bc851dfcc8356d7da3 # pin@v1.5.6
+      - uses: taiki-e/install-action@f860c89ccbfa08ae7bc92502c27ab631f48b8f9d # pin@v2.21.2
         with:
           tool: nextest,cargo-llvm-cov
       - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
-      - run: cargo llvm-cov --ignore-run-fail --package smoke-test --lcov --output-path lcov_smoke.info
+      - run: cargo llvm-cov --ignore-run-fail --package smoke-test --lcov --output-path lcov_smoke.info -vv
```

Comment:
is that -vv intentional?

Reply:
Yes, for debugging and for checking the job status. Without the increased verbosity, we don't know if the cmd is running slowly or getting stuck from the CI perspective.